### PR TITLE
hollaex.fetchMarkets unified structure

### PR DESCRIPTION
**Pull request by samgermain**
```
2022-01-18T00:47:39.711Z
Node.js: v14.17.0
CCXT v1.68.1
hollaex.fetchMarkets ()
572 ms
       id |    symbol | base | quote | settle | baseId | quoteId | settleId | type | spot | margin |  swap | future | option | active | contract | linear | inverse | taker | maker | contractSize | expiry | expiryDatetime | strike | optionType |                         precision |                                                                                          limits
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 xht-usdt |  XHT/USDT |  XHT |  USDT |        |    xht |    usdt |          | spot | true |  false | false |  false |  false |   true |    false |        |         | 0.001 | 0.001 |              |        |                |        |            |     {"price":0.0001,"amount":0.1} |       {"leverage":{},"amount":{"min":0.1,"max":10000000},"price":{"min":0.1,"max":1},"cost":{}}
...
 axs-usdt |  AXS/USDT |  AXS |  USDT |        |    axs |    usdt |          | spot | true |  false | false |  false |  false |   true |    false |        |         | 0.001 | 0.001 |              |        |                |        |            |     {"price":0.01,"amount":0.001} |      {"leverage":{},"amount":{"min":0.01,"max":1000000},"price":{"min":1,"max":1000},"cost":{}}
avax-usdt | AVAX/USDT | AVAX |  USDT |        |   avax |    usdt |          | spot | true |  false | false |  false |  false |   true |    false |        |         | 0.001 | 0.001 |              |        |                |        |            |      {"price":0.1,"amount":0.001} |     {"leverage":{},"amount":{"min":0.001,"max":1000000},"price":{"min":1,"max":1000},"cost":{}}
21 objects
```

### CODE REVIEW

File: js/hollaex.js [hunk 1]
```diff
@@ -189,30 +189,45 @@ module.exports = class hollaex extends Exchange {
...
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': this.safeValue (market, 'active'),
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'taker': this.fees['trading']['maker'],
+                'maker': this.fees['trading']['taker'],
```
Comment (samgermain) [inline js/hollaex.js:216]: Do we want to start adding these fields in to create a more unified structure?

Comment (kroitor) [reply js/hollaex.js:235]: Yes, we do, but we don't want those properties to be hardcoded, if possible.

Comment (kroitor) [reply js/hollaex.js:235]: The markets are extended with `fees` in the base method `setMarkets`, so we do't have to add them here.

Comment (samgermain) [reply js/hollaex.js]: > The markets are extended with `fees` in the base method `setMarkets`, so we do't have to add them here.

Unless I code these  keys in, the output of fetchMarkets doesn't include it, look at `digifinex` for example

Comment (kroitor) [reply js/hollaex.js]: @samgermain fetchMarkets doesn't, but loadMarkets does. It is done in the base class in the `setMarkets` helper.

**Pull request merged**